components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -285,6 +285,8 @@ export namespace GitpodServer {
     }
     export interface CreateWorkspaceOptions {
         contextUrl: string;
+        branch?: string;
+        projectId?: string;
```

Comment:
Leftover? To me this doesn't make sense, and is not actually used anywhere in this PR -- remove for now?

```suggestion
```

Reply:
Indeed.

Reply:
done